README.md

```diff
@@ -1,4 +1,4 @@
-# drone-plugin-index
+# Drone plugin index
```

Comment:
Please keep the headline as before, it's written like that for most drone repos.